app/assets/javascripts/discourse/app/components/user-menu/messages-list.js

```diff
@@ -98,14 +97,9 @@ export default class UserMenuMessagesList extends UserMenuNotificationsList {
     return content;
   }
 
-  dismissWarningModal() {
-    const modalController = showModal("dismiss-notification-confirmation");
-    modalController.set(
-      "confirmationMessage",
-      I18n.t("notifications.dismiss_confirmation.body.messages", {
-        count: this.#unreadMessagesNotifications,
-      })
-    );
-    return modalController;
+  dismissConfirmationText() {
```

Comment:
This feels like it should be a getter as well - it doesn't have any side effects